src/lib/learn-client/types.ts

```diff
@@ -260,6 +265,21 @@ export enum CompanyLogoOption {
 	aws = 'aws',
 }
 
+export enum TutorialProgressLabel {
+	visited = 'visited',
+	in_progress = 'in_progress',
+	complete = 'complete',
+}
+
+export enum TutorialProgressPercent {
+	Zero = '0',
+	TwentyFive = '25',
+	Fifty = '50',
+	SeventyFive = '75',
+	Ninety = '90',
+	OneHundred = '100',
```

Comment:
Any reason we're using a different casing here?

Reply:
No strong reason here, leaned towards matching learn-api [which uses this casing](https://github.com/hashicorp/learn-api/blob/64be7e42bb8852c7c777d60e6732b908fc3042a5/src/models/collection_tutorial_progress.ts#L11).

Going to merge this as-is, but definitely happy to adjust in #900 if we want. I guess for consistency with `dev-dot` we'd want camel case like `seventyFive` etc? Or maybe snake like `seventy_five`? Seeing both in the above types so on the fence.